pages/api/search/[fragment].ts

```diff
@@ -0,0 +1,25 @@
+import axios from "axios"
+
+const BASE_URL = "https://cloud.iexapis.com"
+const TOKEN = "pk_63509e5b43384ab08845be28759fb5ea"
```

Comment:
In a future PR, let's set these as ENV variables.  When we do that, I will update the token.